### DISCUSSION TITLE
Excessive "Downloading https://formulae.brew.sh/api/formula.jws.json"

### DISCUSSION THREAD

**Discussion by cilia** [Homebrew/discussions#4888] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.19
ORIGIN: https://github.com/Homebrew/brew
HEAD: 7f745d04d90eaec76c8d5830d6ce6a0d7db888b3
Last commit: 2 days ago
Core tap JSON: 08 Nov 05:08 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.42.1 => /opt/homebrew/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.6.1-arm64
CLT: 15.0.0.0.1.1694021235
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  openssl@1.1
```

### Description of issue

It seems homebrew now does "Downloading https://formulae.brew.sh/api/formula.jws.json" noticeably more often, whenever I do `brew upgrade` or `brew cleanup`, even if it's downloaded just a couple of hours ago. Is this normal behavior, or something is off?

**Comment by SMillerDev** [comment, marked as answer]
No, that is intended behaviour. It needs to know the latest state of all formulae for most operations. Especially `brew upgrade`

**Reply by cilia** [reply]
Ok, thanks for confirming.

**Comment by emmabasseyokon** [comment]
How do i upgrade  openssl@1.1

**Reply by gromgit** [reply]
No different that upgrading any other Homebrew package:
* `brew update` to make sure you have the latest list of Homebrew packages, then 
* `brew upgrade openssl@1.1`.

If you're _really_ asking "how do I upgrade to a 1.1 release newer than 1.1.1w", then the answer is "you can't". From [OpenSSL's own release strategy page](https://openssl-library.org/policies/releasestrat/index.html):

> Versions 1.1.1 and 1.0.2 are no longer supported

Indeed, no release version below 3.x is supported any longer.
